src/node-server/nlp/src/index.ts

```diff
@@ -0,0 +1,84 @@
+import { config } from "dotenv";
+import { interval, Observable } from "rxjs";
+import {
+  filter,
+  map,
+  mapTo,
+  shareReplay,
+  switchMap,
+  takeUntil
+} from "rxjs/operators";
+import uuid from "uuid/v1";
+import logger from "../../shared/src/logger";
```

Comment:
maybe would be nice to set up `shared` in tsconfig `paths`  so could `import { logger } from 'shared'`?

Reply:
Definitely